www/js/typechecker.js

```diff
@@ -94,13 +94,39 @@ define([
 				}
 
 				return {status: false, errorType: errorType};
-			}else if(inputString == lastInput.value || inputString==""){
+			}
+			else{
+
+				return {status: undefined, errorType: undefined};
+			}
+		},
+
+		checkLastInputValue: function(nodeID, lastInput){
```

Comment:
I think this should be renamed "checkLastInitialValue" since it is hard-coded to check the initial value field.